lib/src/r_tree/r_tree.dart

```diff
@@ -51,12 +51,17 @@ class RTree<E> {
 
   // Returns all items whose rectangles overlap the @searchRect
   //  Note: Rectangles that share only a border are not considered to overlap
-  Iterable<RTreeDatum<E>> search(Rectangle searchRect) {
+  Iterable<RTreeDatum<E>> search(Rectangle searchRect,
+      [bool Function(E item) shouldInclude]) {
```

Comment:
Kind of personal preference, but I think named optional parameters are easier to maintain than optional positional parameters

Reply:
I agree 😄  Much. More. Better.